modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/layout-page-templates/display-page-templates/dxp-release-note-template/display-page-template.json

```diff
@@ -3,5 +3,5 @@
 		"className": "com.liferay.asset.kernel.model.AssetCategory"
 	},
 	"defaultTemplate": true,
-	"name": "DXP Release Notes"
+	"name": "DXP Release Note Template"
```

Comment:
rename to `DXP Release Notes Template`. `Notes` being plural to match the rest of the naming